Lib/subprocess.py

```diff
@@ -409,8 +409,9 @@ def check_output(*popenargs, timeout=None, **kwargs):
     decoded according to locale encoding, or by "encoding" if set. Text mode
     is triggered by setting any of text, encoding, errors or universal_newlines.
     """
-    if 'stdout' in kwargs:
-        raise ValueError('stdout argument not allowed, it will be overridden.')
+    for kw in ('stdout', 'check'):
```

Comment:
please add an equivalent `if 'check' in kwargs:` test to `check_call()` and a unittest for that one.

Reply:
I don't think that's necessary. `check_call()` defers to `Popen` (the docs literally say this so it's not an implementation detail) and `Popen` doesn't support a `check=` argument.